unstructured/embed/bedrock.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+import numpy as np
+import boto3
+
+from unstructured.documents.elements import (
+    Element,
+)
+from unstructured.embed.interfaces import BaseEmbeddingEncoder
+from unstructured.ingest.error import EmbeddingEncoderConnectionError
+from unstructured.utils import requires_dependencies
+from langchain.embeddings import BedrockEmbeddings
+
+class BedrockEmbeddingEncoder(BaseEmbeddingEncoder):
+    def __init__(self, aws_access_key_id: str, aws_secret_access_key: str, region_name: str = "us-west-2"):
+        self.aws_access_key_id = aws_access_key_id
+        self.aws_secret_access_key = aws_secret_access_key
+        self.region_name = region_name
+        self.initialize()
+
+    def initialize(self):
+        self.bedrock_client = self.get_bedrock_client()
+
+    def num_of_dimensions(self):
+        return np.shape(self.examplary_embedding)
+
+    def is_unit_vector(self):
+        return np.isclose(np.linalg.norm(self.examplary_embedding), 1.0)
+
+    def embed_query(self, query):
+        return np.array(self.bedrock_client.embed_query(query))
+
+    def embed_documents(self, elements: List[Element]) -> List[Element]:
+        embeddings = [np.array(self.embed_query(str(e))) for e in elements]
```

Comment:
Will this work? We shouldn't use query embedding while embedding documents
```suggestion
        embeddings = self.bedrock_client.embed_documents([str(e) for e in elements])
```